src/DynamoCore/Extensions/ReadyParams.cs

```diff
@@ -27,5 +27,13 @@ public IEnumerable<IWorkspaceModel> WorkspaceModels
                 return dynamoModel.Workspaces;
             }
         }
+
+        public IWorkspaceModel CurrentWorkspace
```

Comment:
Please rename this to `CurrentWorkspaceModel`.  This will help to distinguish it from a future potential property on the same type, `CurrentWorkspaceViewModel.`